MoneySplitter.Models/Models/Transaction.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace MoneySplitter.Models.Models
```

Comment:
Looks like you added folder Models to Models project. Better name it more informatically. For example Api, in this case we will have Models.Api namespace, it looks more nicely

Reply:
rename Models to Entities